pydm/widgets/archiver_time_plot.py

```diff
@@ -715,9 +730,9 @@ def updateXAxis(self, update_immediately: bool = False) -> None:
         max_x = self.plotItem.getAxis("bottom").range[1]
         max_point = max([curve.max_x() for curve in self._curves])
         if min_x == 0:  # This is zero when the plot first renders
-            min_x = time.time()
-            self._min_x = min_x
-            self._starting_timestamp = min_x - DEFAULT_TIME_SPAN  # A bit of a buffer so we don't overwrite live data
+            self._max_x = time.time()
+            self._min_x = self._max_x - DEFAULT_TIME_SPAN
+            self._starting_timestamp = self._max_x
             if self.getTimeSpan() != DEFAULT_TIME_SPAN:
```

Comment:
After trying this out and looking for why my test plot didn't make the archive call, I realized it's this `if self.getTimeSpan() != DEFAULT_TIME_SPAN:` line (at least on my side). The intention here was just to only make this initial data call if the time span was not the original default of 5 seconds. Now that the default has been changed to 3600, it sould probably be changed to `MIN_TIME_SPAN` instead of `DEFAULT_TIME_SPAN` and then the initial data call would work fine for my use cases without adding the additional one below.

(It was you fixing the default span not being set up above in line 721 that uncovered this bug, so thank you for helping me hunt this down!)